File: enmap/object/manga_info.py
from dataclasses import dataclass
import typing

from object.chapter_info import Chapter_Info

@dataclass
class Manga_Info:
    title: str
    xid: str
    author: str
    status: str
    cover: str
    type: typing.List[str] #
    chapter_nums: int
    chapters_info: typing.List[Chapter_Info]
    description: str

File: server/object/manga_info.go
package object

// json decoder
type MangaInfo struct {
	Title        string `json:"title"`
	Xid			 string `json:"xid"`
	Author 		 string `json:"author"`
	Status 		 string `json:"status"`
	Cover        string `json:"cover"`
	Type         []string `json:"type"`
	Chapter_nums int    `json:"chapter_nums"`
	Description  string `json:"description"`
}